.eslintrc.json

```diff
@@ -45,6 +52,151 @@
+    "@typescript-eslint/restrict-template-expressions": "off",
+    "@typescript-eslint/sort-type-constituents": "off",
+    "@typescript-eslint/strict-boolean-expressions": "off",
+    "@typescript-eslint/switch-exhaustiveness-check": "off",
+    "@typescript-eslint/triple-slash-reference": "off",
+    "@typescript-eslint/typedef": "off",
+    "@typescript-eslint/unbound-method": "off",
+    "@typescript-eslint/unified-signatures": "off",
+
+    // Extension Rules
+    "@typescript-eslint/default-param-last": "off",
+    "@typescript-eslint/dot-notation": "off",
+    "@typescript-eslint/init-declarations": "off",
+    "@typescript-eslint/no-array-constructor": "off",
+    "@typescript-eslint/no-dupe-class-members": "off",
+    "@typescript-eslint/no-duplicate-imports": "off",
+    "@typescript-eslint/no-empty-function": "off",
+    "@typescript-eslint/no-extra-semi": "off",
+    "@typescript-eslint/no-implied-eval": "off",
+    "@typescript-eslint/no-invalid-this": "off",
+    "@typescript-eslint/no-loop-func": "off",
+    "@typescript-eslint/no-loss-of-precision": "off",
+    "@typescript-eslint/no-magic-numbers": "off",
+    "@typescript-eslint/no-redeclare": "off",
+    "@typescript-eslint/no-restricted-imports": "off",
+    "@typescript-eslint/no-shadow": "off",
+    "@typescript-eslint/no-throw-literal": "off",
+    "@typescript-eslint/no-unused-expressions": "off",
+    "@typescript-eslint/no-unused-vars": "off",
+    "@typescript-eslint/no-use-before-define": "off",
+    "@typescript-eslint/no-useless-constructor": "off",
+    "@typescript-eslint/require-await": "off",
+    "@typescript-eslint/return-await": "off",
+
+    // Formatting Rules
+    "@typescript-eslint/block-spacing": "off",
+    "@typescript-eslint/brace-style": "off",
+    "@typescript-eslint/comma-dangle": "off",
+    "@typescript-eslint/comma-spacing": "off",
+    "@typescript-eslint/func-call-spacing": "off",
+    "@typescript-eslint/indent": "off",
+    "@typescript-eslint/key-spacing": "off",
+    "@typescript-eslint/keyword-spacing": "off",
+    "@typescript-eslint/lines-between-class-members": "off",
+    "@typescript-eslint/member-delimiter-style": "off",
+    "@typescript-eslint/no-extra-parens": "off",
+    "@typescript-eslint/object-curly-spacing": "off",
+    "@typescript-eslint/padding-line-between-statements": "off",
+    "@typescript-eslint/quotes": "off",
+    "@typescript-eslint/semi": "off",
+    "@typescript-eslint/space-before-blocks": "off",
+    "@typescript-eslint/space-before-function-paren": "off",
+    "@typescript-eslint/space-infix-ops": "off",
+    "@typescript-eslint/type-annotation-spacing": "off",
+
+    // Deprecates Rules
+    "@typescript-eslint/no-duplicate-imports": "off",
+    "@typescript-eslint/no-implicit-any-catch": "off",
+    "@typescript-eslint/no-parameter-properties": "off",
+    "@typescript-eslint/sort-type-union-intersection-members": "off",
+    "@typescript-eslint/camelcase": "off"
```

Comment:
Did you just turn off all the typescript eslint rules there are in existence? Why would you do that? Shouldn't you only turn off rules which are causing the errors and warnings?